### ISSUE TITLE
Redline (328460)

### ISSUE THREAD

**Issue by shoober420** [ValveSoftware/Proton#3287]
# Compatibility Report
- Name of the game with compatibility issues: Redline
- Steam AppID of the game: https://steamdb.info/app/328460/

## System Information
- GPU: GTX 760
- Driver/LLVM version: NVIDIA 440.36
- Kernel version: 5.4.1
- https://github.com/shoober420/linux-scripts/blob/main/home/shoober420/sysinfonvidia
- Proton version: 4.11-9

## Symptoms
Explosions and gibs have black squares around the sprites.

## Reproduction
Generate explosions and gibs

The black squares are gone and are rendered correctly when using dgVoodoo2 (http://dege.freeweb.hu/dgVoodoo2/dgVoodoo2.html).

**Comment by shoober420** [comment]
Here is a log of the game running without dgVoodoo, producing the black squares around gibs and particles.

[steam-328460.log](https://github.com/ValveSoftware/Proton/files/3993264/steam-328460.log)

**Comment by kisak-valve** [comment]
These look like lines of interest from the log:
```
err:ddraw:d3d_texture2_Load Trying to load surfaces with different mip-map counts.
err:ddraw:d3d_texture2_Load Loading surface with different mipmap structure.
```

Possibly related: https://github.com/wine-staging/wine-staging/tree/master/patches/ddraw-IDirect3DTexture2_Load

**Comment by Arucard1983** [comment]
I agree that dgVoodoo2 should be added to Proton, maybe on a special third-party folder to avoid mixing of closed and open-sourced components.
